packages/web-components-config-server/src/routes/web-components-config/web-components-config.ts

```diff
@@ -67,9 +54,26 @@ export const webComponentsConfigDeleteHandler = async (req: AppRequest, res: App
   }
 }
 
-webComponentsConfig.get('/:customerId', webComponentsConfigGetByIdHandler)
-webComponentsConfig.put('/', webComponentsConfigPutHandler)
-webComponentsConfig.patch('/:customerId', webComponentsConfigPatchHandler)
-webComponentsConfig.delete('/:customerId', webComponentsConfigDeleteHandler)
+export const webComponentsConfigPostHandler = async (req: AppRequest, res: AppResponse) => {
+  try {
+    const params = {
+      data: { ...req.body, customerId: req.params.customerId, appId: req.params.appId },
```

Comment:
shoud we use opt chain to check null or undefined ?

Reply:
It's params from express route matching. Pretty sure about value of these field are always something not null